test/e2e/helper_test.go

```diff
@@ -0,0 +1,332 @@
+// Copyright 2021 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//go:build e2e || kubernetes
+
+package e2e
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/url"
+	"os"
+	"os/exec"
+	"strings"
+
+	"github.com/mongodb/atlas-cli-plugin-kubernetes/internal/kubernetes/operator/resources"
+	"github.com/mongodb/atlas-cli-plugin-kubernetes/test"
+	atlasv2 "go.mongodb.org/atlas-sdk/v20241113004/admin"
+	"go.mongodb.org/atlas/mongodbatlas"
+)
+
+const (
+	clustersEntity = "clusters"
+	projectEntity  = "project"
+	orgEntity      = "org"
+)
+
+// Cluster settings.
```

Comment:
I am of the opinion that code should not have comments that don't bring any value
```suggestion
```